### ISSUE TITLE
Service discovery does not work on 14.1

### ISSUE THREAD

**Issue by dstokesf5** [F5Networks/f5-appsvcs-extension#70]
##### ISSUE TYPE
 - Bug Report

##### AS3 BUILD/ VERSION
```
3.9.0
```

##### BIGIP VERSION
```
Sys::Version
Main Package
  Product     BIG-IP
  Version     14.1.0.1
  Build       0.0.7
  Edition     Point Release 1
  Date        Tue Jan 15 19:49:59 PST 2019
```

##### SUMMARY
```
The service discovery worker used by AS3 does not handle the encryption related API changes in 14.1 and cannot properly handle credentials. This prevents the service discovery from working on 14.1
```

##### STEPS TO REPRODUCE
```json
{
	"class": "ADC",
	"schemaVersion": "3.9.0",
	"id": "Pool",
	"TEST_ServiceDiscovery": {
		"class": "Tenant",
		"Application": {
			"class": "Application",
			"template": "generic",
			"testItem": {
				"class": "Pool",
				"members": [{
						"addressDiscovery": "azure",
						"updateInterval": 60,
						"tagKey": "foo",
						"tagValue": "bar",
						"addressRealm": "private",
						"resourceGroup": "**removed**",
						"subscriptionId": "**removed**",
						"directoryId": "**removed**",
						"applicationId": "**removed**",
						"apiAccessKey": "**removed**",
						"credentialUpdate": false,
						"servicePort": 8080
					}
				]
			}
		}
	}
}

```

##### EXPECTED RESULTS

```
Pool members should be discovered from the provided Azure environment
```

##### ACTUAL RESULTS
```
Pool members are not discovered and errors about being unable to encrypt values that are not secure vault cryptograms appear in the restnoded log.
```

**Comment by dstokesf5** [comment]
This issue has been resolved and the fix will be available in the 3.10 release.

**Comment by kaustriaf5** [comment]
This was tracked internally as 919, fixed in [3.10](https://github.com/F5Networks/f5-appsvcs-extension/releases/tag/v3.10.0)
